astropy/coordinates/tests/accuracy/test_altaz_icrs.py

```diff
@@ -149,7 +152,8 @@ def test_against_jpl_horizons():
     assert distance < 1 * u.arcsec
 
 
-@pytest.mark.xfail
+@pytest.mark.remote_data
+@pytest.mark.xfail(reason="Current output is completely incorrect")
```

Comment:
ouch. any chance to fix this @adrn/@eteq? I mean in a new PR not in this one.

Reply:
I made an issue to remind us #8296